Example/AppDelegate.h

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
These comments will need to be removed and adding the MIT license we use in all other documents.

Reply:
Its a UI to test this solution.